package.json

```diff
@@ -38,80 +38,75 @@
     "prepare": "husky install"
   },
   "dependencies": {
-    "@nodevu/core": "^0.1.0",
-    "@types/node": "^18.16.19",
-    "@vcarl/remark-headings": "^0.1.0",
+    "@nodevu/core": "0.1.0",
+    "@types/node": "18.16.19",
+    "@vcarl/remark-headings": "0.1.0",
     "@vercel/analytics": "^1.0.1",
-    "classnames": "^2.3.2",
-    "cross-env": "^7.0.3",
-    "feed": "^4.2.2",
-    "framer-motion": "^10.12.18",
-    "glob": "^10.3.1",
-    "gray-matter": "^4.0.3",
-    "highlight.js": "^11.8.0",
-    "husky": "^8.0.3",
-    "next": "^13.4.7",
-    "next-mdx-remote": "^4.4.1",
-    "next-sitemap": "^4.1.8",
-    "next-themes": "^0.2.1",
-    "prismjs": "^1.29.0",
-    "react-icons": "^4.10.1",
-    "react-intl": "^6.4.4",
-    "remark-gfm": "^3.0.1",
-    "semver": "^7.5.3",
-    "sharp": "^0.32.1",
+    "classnames": "2.3.2",
+    "cross-env": "7.0.3",
+    "feed": "4.2.2",
+    "framer-motion": "10.12.18",
+    "glob": "10.3.3",
+    "gray-matter": "4.0.3",
+    "highlight.js": "11.8.0",
+    "husky": "8.0.3",
+    "next": "^13.4.9",
+    "next-mdx-remote": "4.4.1",
+    "next-sitemap": "4.1.8",
+    "next-themes": "0.2.1",
+    "prismjs": "1.29.0",
+    "react-icons": "4.10.1",
+    "react-intl": "6.4.4",
+    "remark-gfm": "3.0.1",
+    "semver": "7.5.4",
+    "sharp": "0.32.1",
     "turbo": "^1.10.7",
-    "typescript": "^5.1.6"
+    "typescript": "5.1.6"
```

Comment:
devDependency

Reply:
typescript doesn't follow semver, so `~` is the proper range token to use for it, fwiw

Reply:
In this case it should be a regular dependency, together with `@†ypes/node` otherwise, Next.js complains if we want a minimal install `npm i --omit=dev`